scripts/start_civ.sh

```diff
@@ -569,6 +599,8 @@ check_nested_vt || exit -1
 
 setup_rpmb_dev || exit -1
 setup_swtpm
+set_share_aaf
```

Comment:
Add a new function: `set_default_aaf_config()` which aimed to set all default configs. Execute it before parse_arg(line 595). Then in later functions, just modify the configs or add new configs.